ios/StripeTerminalReactNative.swift

```diff
@@ -17,10 +17,13 @@ enum ReactNativeConstants: String, CaseIterable {
     case START_READER_RECONNECT = "didStartReaderReconnect"
     case READER_RECONNECT_SUCCEED = "didSucceedReaderReconnect"
     case READER_RECONNECT_FAIL = "didFailReaderReconnect"
+    case OFFLINE_STATUS_CHANGE = "didOfflineStatusChange"
+    case PAYMENT_INTENT_FORWARDED = "didPaymentIntentForwarded"
```

Comment:
lets go with `didChangeOfflineStatus` and `didForwardPaymentIntent` to be consistent with rest

Reply:
I left a similar naming comment above. It's not totally clear to me how these are used, so I'll defer to you here.